src/components/SelectionList/CardListItem.tsx

```diff
@@ -77,6 +77,13 @@ function CardListItem<TItem extends ListItem>({
                                 item.alternateText ? styles.mb1 : null,
                             ]}
                         />
+                        {!!item.lastFourPAN && (
+                            <TextWithTooltip
+                                shouldShowTooltip={showTooltip}
+                                text={`Ending in ${item.lastFourPAN}`}
```

Comment:
In the PR description you checked the following checkboxes:

> * [x] I verified any copy / text shown in the product is localized by adding it to src/languages/* files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
>     * [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:

but **did not actually implement translations** for this line. Hopefully, this was just a mistake and you are actually checking the PR Author Checklist boxes thoroughly❗

```suggestion
                                text={translate('subscription.cardSection.cardEnding', {
                                    cardNumber: item.lastFourPAN,
                                })}
```

##

> [!note]
> You might want to **hold** on actually applying this change until design confirms on the issue https://github.com/Expensify/App/issues/47324#issuecomment-2303088140 whether we should add new `Ending in` and translate it or use already existing `Card ending in` which is translated already.